cypress/page-object/components/Header.ts

```diff
@@ -0,0 +1,35 @@
+export default class Header {
+
+  private headerContainer: string = '#navbarExample';
+
+  private get logInButton(): Cypress.Chainable {
+    return cy.get(`${this.headerContainer} [id="login2"]`);
+  }
+  private get logOutButton(): Cypress.Chainable {
+    return cy.get(`${this.headerContainer} [id="logout2"]`);
+  }
+  private get welcomeUserName(): Cypress.Chainable {
+    return cy
+      .get(`${this.headerContainer} [id="nameofuser"]`, { timeout: 5000 })
+      .should(($x) => {
+        expect($x).to.have.text('Welcome SarahConnor');
+      });
+  }
```

Comment:
Add a new row for each end of the getter instruction to make the code more readable!